Update Docs Homepage (#529)
ppb/pursuedpybear: docs/index.rst

Welcome to PursuedPyBear's documentation!
=========================================

PursuedPyBear, also known as :code:`ppb`, exists to be an educational resource. Most obviously used to teach computer science, it can be a useful tool for any topic that a simulation can be helpful. At its core, :code:`ppb` provides a number of features that make it perfect for video games. The :code:`GameEngine` itself provides a pluggable subsystem architecture where adding new features is as simple as subclassing and extending :code:`System`. 

> .. note :: This document explains the practical and technical details of :code:`ppb`. If you want to know more about what it is, its goals, or why you should use it, check out `this page<https://github.com/ppb/pursuedpybear/blob/canon/README.md>` `our website<https://ppb.dev>`.

Reviewer: ```suggestion
.. note :: This document explains the practical and technical details of :code:`ppb`. If you want to know more about what it is, its goals, or why you should use it, check out `this page<https://github.com/ppb/pursuedpybear/blob/canon/README.md>`_ of `our website<https://ppb.dev>`_.
```

Using these docs
----------------

> Our documentation is seperated into practical steps and theroretical knowledge. The docs are made up of Tutorials, which allow you to learn quickly, and guides which bring you to a specific goal. We also have a standard API reference.

Reviewer: ```suggestion
Our documentation is separated into practical steps and theoretical knowledge. The docs are made up of Tutorials, which allow you to learn quickly, and guides which bring you to a specific goal. We also have a standard API reference.
```

As a New User

> ^^^^^^^^^^^^^

Reviewer: ```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Built locally, rst didn't like the line length of the underline. Best practice is to overkill since when we start adding translations, this line must be long enough for all translations.

We recommend that new users start with the :doc:`getting-started` Guide. This document assumes that you have basic programming knowledge. If you don't, check out the Python docs at https://docs.python.org. 

As an Experienced User

> ^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Built locally, rst didn't like the line length of the underline. Best practice is to overkill since when we start adding translations, this line must be long enough for all translations.
